[CI] IndexingIT testIndexing failing

**Build Scans:**
- [elasticsearch-intake #24127 / 8.19.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/f2356jsucupo6)
- [elasticsearch-pull-request #75895 / 8.19.0_part_2_bwc-snapshots](https://gradle-enterprise.elastic.co/s/xn2k5hqbobgoi)

**Reproduction Line:**
```
./gradlew ":x-pack:qa:rolling-upgrade:v8.19.0#upgradedClusterTest" -Dtests.class="org.elasticsearch.upgrades.IndexingIT" -Dtests.method="testIndexing" -Dtests.seed=727871A2E3E80A61 -Dtests.bwc=true -Dtests.locale=lv -Dtests.timezone=Asia/Colombo -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.upgrades.IndexingIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testIndexing),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: timed out waiting for green state for index [test_index,index_with_replicas,empty_index] cluster state [{
  "cluster_name" : "v8.19.0",
  "cluster_uuid" : "G-anRk73RlCyKG51LVCgAQ",
  "version" : 1903,
  "state_uuid" : "v5hnpHUwQbKNwwFLXADpxg",
  "master_node" : "5PBptISmTnCoJB5TRVfoVw",
  "blocks" : {
    "indices" : {
      "mounted_index_shared_cache" : {
        "8" : {
          "description" : "index write (api)",
          "retryable" : false,
          "levels" : [
            "write"
          ]
        }
      },
      ".ds-reindex_test_data_stream_ugprade_test-2025.06.17-000004" : {
        "4" : {
          "description" : "index closed",
          "retryable" : false,
          "levels" : [
            "read",
            "write"
          ]
        }
      },
      "mounted_index_full_copy" : {
        "8" : {
          "description" : "index write (api)",
          "retryable" : false,
          "levels" : [
            "write"
          ]
      
[truncated]
```

**Issue Reasons:**
- [main] 2 failures in test testIndexing (0.2% fail rate in 1000 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -470,9 +470,6 @@ tests:
 - class: org.elasticsearch.xpack.security.PermissionsIT
   method: testWhenUserLimitedByOnlyAliasOfIndexCanWriteToIndexWhichWasRolledoverByILMPolicy
   issue: https://github.com/elastic/elasticsearch/issues/129481
-- class: org.elasticsearch.upgrades.IndexingIT
-  method: testIndexing
-  issue: https://github.com/elastic/elasticsearch/issues/129533
 - class: org.elasticsearch.upgrades.MlJobSnapshotUpgradeIT
   method: testSnapshotUpgrader
   issue: https://github.com/elastic/elasticsearch/issues/98560

